scripts/check-dependencies.js

```diff
@@ -0,0 +1,17 @@
+var pkg = require('../package.json');
```

Comment:
I had removed this in a past commit since I moved it to pact-mock-service as a wrapper.  Should probably do the same with pact-provider-verifier.